web/api/js/codechecker-api-node/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "codechecker-api",
-  "version": "6.39.0",
+  "version": "6.41.0",
```

Comment:
The previous version number is 6.40.0, but on the left side of this git change I can see 6.39.0. Did you rebase your branch?

Reply:
Yes, I have done it now 😄